Makefile.release

```diff
@@ -145,7 +145,7 @@ else
 	    mkdir -p build/docker/linux/$${arch} ;\
 	    tar -xzf release/$(NAME)_$(VERSION)_linux_$${arch}.tgz -C build/docker/linux/$${arch} ;\
 	    cp Dockerfile build/docker/linux/$${arch} ;\
-	    docker build -t $(DOCKER_IMAGE_NAME)-$${arch}:$(VERSION) build/docker/linux/$${arch} ;\
+	    DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --load --platform linux/$${arch} -t $(DOCKER_IMAGE_NAME)-$${arch}:$(VERSION) build/docker/linux/$${arch} ;\
```

Comment:
why does this need this experimental stuff? I don't trust docker to do the right thing and this has bitten us before. Why does an image change require these changes as well?

I think about 3 released failed because docker when fubar, and I'm close to ripping it all out because of the fragility of it all

Reply:
It doesn't need the experimental stuff I think it only needs DOCKER_BUILDKIT=1. I know k8s switched to using it so I thought we can switch it too. 

> I think about 3 released failed because docker when fubar, and I'm close to ripping it all out because of the fragility of it all
What is the "it" that you plan to rip out?

Reply:
We need the experimental stuff that because:
1. It lets us run part of the build stage in the arch of the machine building the image
2. It preserves xattrs so capabilities applied to the binary in one build-stage remain if the binary is copied to the other build stage. 
3. FWIW k8s switched all their builds to it.

> I think about 3 released failed because docker when fubar, and I'm close to ripping it all out because of the fragility of it all
What is the "it" that you are close to ripping out?

Reply:
The explanation that probably makes sense, see https://github.com/kubernetes/kubernetes/pull/87954

But IMO, I don't think it makes too much sense to enable buildx. 😄

Reply:
without buildx you cannot have xattrs in a dockerfile build. the image format supports it, but not "docker build". buildx does.